queries.sql

```diff
@@ -133,4 +133,68 @@ GROUP BY owners.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
 
+/* Queries for visits, specializations and vets */ 
+SELECT animals.name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Vet William Tatcher'
+ORDER BY visits.visit_date DESC
+LIMIT 1;
+
+SELECT COUNT(DISTINCT visits.animal_id)
+FROM visits
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Vet Stephanie Mendez';
+
+SELECT vets.name, species.name AS specialty
+FROM vets
+LEFT JOIN specializations ON vets.id = specializations.vet_id
+LEFT JOIN species ON specializations.species_id = species.id;
+
+SELECT animals.name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Vet Stephanie Mendez'
+      AND visits.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
```

Comment:
- [ ] Stephanie Mendez visited `Agumon and Blossom` between these two dates but your query resulted in `0 rows returned` so kindly consider fixing it.